app/action.rb

```diff
@@ -0,0 +1,7 @@
+# auto_register: false
```

Comment:
Is there a specific reason this action.rb class is in "app/" and not in "app/actions"

Reply:
This is default by hanami. I think it prevent that module to be auto-registered by app. We include that module in others actions.